web/src/main/java/org/akaza/openclinica/controller/helper/MessageLogger.java

```
@@ -100,6 +103,70 @@ public void writeToLog(String subDir,String orginalFileName, String headerLine,S
 	    }
 	    
 	   	    
+	} 
+    
+ public void writeToLog(String subDir,String orginalFileName, String headerLine,String msg,UserAccountBean userBean) {
+		
+    	BufferedWriter bw = null;
+		FileWriter fw = null;
+		boolean isNewFile = false;
+
+	    String logFileName;
+	   
+	    if(orginalFileName == null) {
+	    	;
+	    }else {
+	    	 try {
+	             int count =1;	    	
+	 	    		    	
+	 	    	File logFile;
+	 	    	String importFileDir = this.getPersonalLogFileDir(userBean,subDir);
+	     	    
+	 	    	logFileName = importFileDir + orginalFileName;
+	 			logFile = new File(logFileName);
+	 			
+	 			/**
+	 			 *  create new file and add headerLine as first line
+	 			 *  example :
+	 			 *  RowNo | ParticipantID | Status | Message
+	 			 */
+	 			if(!logFile.exists()) {
+	 				logFile.createNewFile();
+	 				isNewFile = true;				
+	 			}
+	 			
+	 			// true = append file
+	 			fw = new FileWriter(logFile.getAbsoluteFile(), true);
+	 			bw = new BufferedWriter(fw);
+	            
+	 			// create new file and prepare header line
+	 			if(isNewFile) {				
+	 				bw.write(headerLine);	
+	 				bw.write("\n");
+	 			}
+	 			
+	 			if(msg != null) {
+	 				bw.write(msg);	
+	 				bw.write("\n");
+	 			}	
+	 			
+	 			bw.close();						
+	 	       
+	 	    } catch (Exception e) {
+	 	        e.printStackTrace();
```

Do not print the trace. Log it. 
Do not eat the exception. Throw it back.